InnerEye/ML/configs/other/fastmri_varnet.py

```diff
@@ -37,32 +43,131 @@ def log_image(self, name: str, image: torch.Tensor) -> None:
                 experiment.add_image(name, image, global_step=self.global_step)
 
 
+def get_data_module(azure_dataset_id: str,
```

Comment:
Would it make sense to name that get_fastmri_data_module?

Reply:
yup.